dotenv.cabal

```diff
@@ -100,7 +100,7 @@ library
   build-depends:         base >=4.7 && <5.0
                        , base-compat >= 0.4
                        , directory
-                       , megaparsec >= 7.0.1 && < 9.0
+                       , megaparsec >= 7.0.1 && < 9.1
```

Comment:
Would it be better to enable it for `< 10`? So that you don't have to pump version dependencies for nonbreaking changes